gazelle/python/generate.go

```diff
@@ -54,6 +55,24 @@ func GetActualKindName(kind string, args language.GenerateArgs) string {
 	return kind
 }
 
+func makeCompiledGlobs(globs []string) []glob.Glob {
+	compiledGlobs := []glob.Glob{}
+	for _, value := range globs {
+		compiledGlob := glob.MustCompile(value)
```

Comment:
I might prefer a more helpful error message if the glob fails. Could we add an integration test? Maybe it is good enough already? Right now this code would panic